src/argilla/client/feedback/schemas.py

```diff
@@ -0,0 +1,157 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import warnings
+from typing import Any, Dict, List, Optional, Union
+from uuid import UUID
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+from pydantic import (
+    BaseModel,
+    Extra,
+    Field,
+    StrictInt,
+    StrictStr,
+    validator,
+)
+
+FETCHING_BATCH_SIZE = 250
+PUSHING_BATCH_SIZE = 32
+
+
+class ValueSchema(BaseModel):
+    value: Union[StrictStr, StrictInt]
+
+
+class ResponseSchema(BaseModel):
+    user_id: Optional[UUID] = None
+    values: Dict[str, ValueSchema]
+    status: Literal["submitted", "discarded"] = "submitted"
```

Comment:
In #3033, which has been already merged to `develop`, we added a new possible status called `draft`. It's not being used yet by the UI, but I think that we can add it here already.

Reply:
My take here is that we shouldn't confuse the user with "useless" stuff, so maybe we can leave it as is until we decide what to do with the "draft" status?

Reply:
okay! we can add it later in another PR